onboard/src/pages/subsections/backgroundInformation/Personalnformation.jsx

```diff
@@ -1,63 +1,112 @@
+import React from "react";
 import TextField from "@mui/material/TextField";
+import { styled } from "@mui/material/styles";
+import Grid from "@mui/material/Grid";
+import Paper from "@mui/material/Paper";
+import Box from "@mui/material/Box";
 
-function PersonalInformation () {
-    return (
-        <>
-            <TextField
-              id="outlined-basic"
-              label="First Name"
-              variant="outlined"
-            />
+const Item = styled(Paper)(({ theme }) => ({
+  backgroundColor: theme.palette.mode === "dark" ? "#1A2027" : "#fff",
+  ...theme.typography.body2,
+  padding: theme.spacing(1),
+  textAlign: "center",
+  color: theme.palette.text.secondary,
+}));
 
-            <TextField
-              id="outlined-basic"
-              label="Last Name"
-              variant="outlined"
-            />
-
-            <TextField
-              id="outlined-basic"
-              label="Middle Initial"
-              variant="outlined"
-            />
-
-            <TextField
-              type="number"
-              id="outlined-basic"
-              label="SSN"
-              variant="outlined"
-            />
-            <TextField
-              type="date"
-              id="outlined-basic"
-              label="Birthdate"
-              variant="outlined"
-            />
-            <TextField
-              id="outlined-basic"
-              label="Current Address"
-              variant="outlined"
-            />
-            <TextField
-              id="outlined-basic"
-              label="Permanent Address"
-              variant="outlined"
-            />
-            <TextField
-              type="number"
-              id="outlined-basic"
-              label="Primary Phone Number"
-              variant="outlined"
-            />
-            <TextField
-              type="number"
-              id="outlined-basic"
-              label="Secondary Phone Number"
-              variant="outlined"
-            />
-          </>
-    );
+function PersonalInformation() {
+  return (
+    <>
+      <Box sx={{ width: "100%" }}>
+        <Grid container rowSpacing={1} columnSpacing={{ xs: 1, sm: 2, md: 3 }}>
+          <Grid item xs={4}>
+            <Item>
+              <TextField
+                id="outlined-basic"
+                label="First Name"
+                variant="outlined"
+              />
+            </Item>
+          </Grid>
+          <Grid item xs={4}>
+            <Item>
+              <TextField
+                id="outlined-basic"
+                label="Last Name"
+                variant="outlined"
+              />
+            </Item>
+          </Grid>
+          <Grid item xs={4}>
+            <Item>
+              <TextField
+                id="outlined-basic"
+                label="Middle Initial"
+                variant="outlined"
+              />
+            </Item>
+          </Grid>
+          <Grid item xs={4}>
+            <Item>
+              <TextField
```

Comment:
Add some controllers that check using regex whether the appropriate sort of information has been entered into fields. Use a debounce function so it does not fire every millisecond and slow the process down.

Reply:
Also, make SSN a password type field so they can't see the info after it has been entered. This field (after it has been submitted the first time) should not allow a user to view the data entered, unlike other fields. DOB should be the same in my opinion.